pkg/skaffold/deploy/kustomize.go

```diff
@@ -114,9 +119,20 @@ func NewKustomizeDeployer(runCtx *runcontext.RunContext) *KustomizeDeployer {
 		BuildArgs:          runCtx.Cfg.Deploy.KustomizeDeploy.BuildArgs,
 		globalConfig:       runCtx.Opts.GlobalConfig,
 		addSkaffoldLabels:  runCtx.Opts.AddSkaffoldLabels,
+		useKubectl:         useKubectl,
 	}
 }
 
+// Check for existence of kustomize binary in user's PATH
+func kustomizeBinaryExists() bool {
+	cmd := exec.Command("which", "kustomize")
+	if err := util.RunCmd(cmd); err != nil {
+		return false
+	}
+
+	return true
```

Comment:
might also be worth logging the version of the binary to the user.

Reply:
What if they have an old old version of `kustomize`?

Reply:
i read this PR as if users have kustomize binary use it even if its older version. 
Does not matter if they have higher version of kubectl 1.14.
it would be really hard for us to make sure the flags users have specified in the `deploy.Kustomize.Flags` config are for `kustomize` or `kubectl kustomize`